tests/helper/tests/wireguard.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import helper.utils as utils
+
+logger = logging.getLogger(__name__)
+
+def wireguard(client):
+    """Validate if the wirefuard kernel module is present"""
+    # Get running kernel version
+    kernel_ver = utils.get_kernel_version(client)
+
+    # Create path to wireguard kernel module
+    kernel_mod_path = f"/lib/modules/{kernel_ver}/kernel/drivers/net/wireguard/"
+    kernel_mod_file = "wireguard.ko"
+    kernel_wireguard_path = f"{kernel_mod_path}{kernel_mod_file}"
```

Comment:
I would suggest to parameterize this tests, so that we can re-use it for other modules as well.

Reply:
Goot point... done